### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by gpacix**
Many minor documentation fixes
Added some syntax checking to the values in glue.auto.tfvars

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ There are some required configurations and setup that must be completed before r
     Access to an AWS account is required.  Please contact `help@excella.com` to gain access to the ExcellaLabs account.  Regardless of the account, an `access_key` and `secret_key` are required for programatic access to the environment.   
 1. AWS CLI install and Configuration  
     Once access to an AWS account is available, as well as an `access_key` and `secret_key`, the [AWS CLI Tool](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) must be installed, and configured by running `aws configure`  
-1. Configure the AWS CLI when using MFA
+1. Configure the AWS CLI when using MFA [why aren't we just having them use `awsmfa`?]
     Special consideration is required when your AWS account is protected with a MFA device.  To configure the CLI, a `token` is required, which will provide temporary credentials.
```
Comment (Wyntuition) [inline README.md]: for now you could put (or use awsmfa, <link>)

Comment (Wyntuition) [reply README.md]: and put a link

File: README.md [hunk 2]
```diff
@@ -69,15 +68,15 @@ Follow the below steps to deploy the following into a clean account. This will b
 * Sonar instance
 
 1. Generate and configure repos (default includes front-end, API, IaC for Jenkins and ECS. If you're feeling bold you can try the *experimental* approach below this.)
-    1. Fork, clone and cd into https://github.com/excellaco/tcp-glue
+    1. Fork, clone and cd into https://github.com/excellaco/tcp-glue [Why do they need to fork it? They won't be changing tcp-glue.]]
```
Comment (Wyntuition) [inline README.md]: Correct, they don't need to fork. If using xg, it creates the repos with new names, which is what we want (whether using xg or not). So if not using xg, after cloning they are going to have to rename and point them to new githubs. What do you think about going full on xg?

File: README.md (hunk 1 above)
Comment (gpacix) [reply README.md]: Added steps for using awsmfa.

File: README.md (hunk 2 above)
Comment (gpacix) [reply README.md]: OK; I'll remove the comment. I think xg-only at some point might be a worthwhile goal.

**Pull request merged**